src/content/docs/integrations/amazon-integrations/aws-integrations-list/aws-metric-stream.mdx

```diff
@@ -92,7 +86,8 @@ We provide a [CloudFormation template](https://console.aws.amazon.com/cloudforma
   * Ensure the following settings are defined:
     * Third-party service provider: New Relic - Metrics
     * New Relic configuration
-      * HTTP endpoint URL: `https://aws-api.newrelic.com/cloudwatch-metrics/v1`
+      * HTTP endpoint URL (US Datacenter): `https://aws-api.newrelic.com/cloudwatch-metrics/v1`
+      * HTTP endpoint URL (EU Datacenter): `https://aws-api.newrelic.com/cloudwatch-metrics/v1`
```

Comment:
@josemore, the URL is the same for both datacenters, is that correct?

Reply:
good catch! EU it's actually: https://cloud-collector.eu01.nr-data.net/cloudwatch-metrics/v1

Reply:
updated